.golangci.yml

```diff
@@ -41,6 +41,12 @@ linters:
   # trigger funlen problems that we may not want to solve at that time.
   - funlen
 
+  - gocyclo
```

Comment:
Why is this disabled?

Reply:
It failed the build due to a code movement. We also haven't calibrated it at all for Go and our codebase in general. I can see new contributors being discourage when they add a small change (say typo fix or minor code movement), but then the build fails due to existing code exceeding this value. I'm down to enable it if we can calibrate it properly, but that seems outside the scope of this PR.

Reply:
Add rationale as a comment perhaps?